### DISCUSSION TITLE
Integration with Outline?

### DISCUSSION THREAD

**Discussion by kenske** [umputun/remark42#1364] [General]
Has anyone integrated Remark42 with [Outline](https://github.com/outline/outline)? That KB is missing a commenting system and it seems that Remark42 would be a great solution.

**Comment by paskal** [comment]
Hi @tommoor, what do you think about it?

**Reply by tommoor** [reply]
We're currently developing comments in Outline tightly integrated with the editor and API. So it's "coming soon" 😄
